detectlanguage_test.go

```diff
@@ -0,0 +1,23 @@
+package cohere
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDetectLanguage(t *testing.T) {
+	co, err := CreateClient(apiKey)
+	if err != nil {
+		t.Error(err)
+	}
+
+	t.Run("DetectLanguage success", func(t *testing.T) {
```

Comment:
I think we need to add an invalid request test too, maybe something basic like an empty string

Reply:
We are not doing any special error handling, so I don't think there is anything to test with that